resources-generator/src/main/kotlin/dev/icerock/gradle/generator/js/AndroidParamsHelper.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.gradle.generator.js
+
+private val androidParamRegex = "%(.)(?:\\\$(.))?".toRegex()
+
+fun String.replaceAndroidParams(): String {
```

Comment:
why it's named android params?

Reply:
I simply kept it the same way it was done in the other code. I realized that the code may have bugs in it, but that seems to have been fixed in this [commit](https://github.com/Intektor/moko-resources/commit/db3386634b54091b620d1a3bc19c90ce61def98e). So it can be changed to the fixed version now.

Reply:
rename to `convertToMessageFormat`